conda/resolve.py

```diff
@@ -849,6 +849,19 @@ def mysat(specs, add_if=False):
             constraints = r2.generate_spec_constraints(C, specs)
             return C.sat(constraints, add_if)
 
+        # Return a solution of packages
+        def clean(sol):
+            return [q for q in (C.from_index(s) for s in sol)
+                    if q and q[0] != '!' and '@' not in q]
+
+        # Determine if there are multiple "simple" solutions
```

Comment:
Can you expand on what a "simple" solution is

Reply:
'simple' or 'obvious' solutions are defined as solution where none of the packages in
the current best solution are used.  I should add this to the comment.